rust/src/cli/ncl.rs

```diff
@@ -193,6 +193,14 @@ fn main() {
                         .takes_value(false)
                         .help("Do not make the state persistent"),
                 )
+                .arg(
+                    clap::Arg::new("SKIP_MERGE")
+                        .long("skip-merge")
+                        .takes_value(false)
+                        .help(
+                            "Do not merge current network state"
```

Comment:
This help text is not sufficient to understand what this option does.

Reply:
This is a brief message, I have added manpage with detail for this.